### PR TITLE
Update File Control

### PR DESCRIPTION
**Pull request by TheMonDon**
**Please describe the changes this PR makes and why it should be merged:**
Fix all the mistakes in file control with changing it to fs-extra

**Status**

- [X] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [X] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Review by almeidx — CHANGES_REQUESTED** [review]
I would like if you addressed these issues as well:

- [ ] `fs.writeFileSync` does not take a callback
https://github.com/dbm-network/mods/blob/a08bbcba8a0900e7f3e7cacb438bf0848c332eda/actions/file_control_MOD.js#L232
https://github.com/dbm-network/mods/blob/a08bbcba8a0900e7f3e7cacb438bf0848c332eda/actions/file_control_MOD.js#L239
https://github.com/dbm-network/mods/blob/a08bbcba8a0900e7f3e7cacb438bf0848c332eda/actions/file_control_MOD.js#L243

- [ ] `fs.appendFileSync` does not take a callback
https://github.com/dbm-network/mods/blob/a08bbcba8a0900e7f3e7cacb438bf0848c332eda/actions/file_control_MOD.js#L251
https://github.com/dbm-network/mods/blob/a08bbcba8a0900e7f3e7cacb438bf0848c332eda/actions/file_control_MOD.js#L255

- [ ] Use `fs.unlinkSync` instead
https://github.com/dbm-network/mods/blob/a08bbcba8a0900e7f3e7cacb438bf0848c332eda/actions/file_control_MOD.js#L267

- [ ] Use `fs.copySync` instead
https://github.com/dbm-network/mods/blob/a08bbcba8a0900e7f3e7cacb438bf0848c332eda/actions/file_control_MOD.js#L273

File: actions/file_control_MOD.js [hunk 1]
```diff
@@ -226,56 +245,48 @@ module.exports = {
...
-      default:
-        break
+          break
+        case 3: // Delete File
+          fs.unlinkSync(fpath)
+          if (!fs.existsSync(dirName)) return this.callNextAction(cache)
+          break
+        case 5: // Copy File
+          fs.copySync(fpath, fpath2)
+          // return console.error(`${lmg} copying: [${err}]`)
```
Comment (almeidx) [inline actions/file_control_MOD.js:282]:
```suggestion
```

File: actions/file_control_MOD.js [hunk 2]
```diff
@@ -10,167 +10,186 @@ module.exports = {
...
-  </div>
-</div>`
+  </div>`
   },
 
   init () {
-    const { document } = this
+    const {
+      document
+    } = this
```
Comment (almeidx) [inline actions/file_control_MOD.js:192]: Revert

**Review by DetroitWhiskey136 — APPROVED** [review]

File: actions/file_control_MOD.js [hunk 3]
```diff
@@ -226,61 +243,51 @@ module.exports = {
...
+          if (fileName === '') break
+          fs.writeFileSync(fpath, '')
+          break
+        case 1: // Write File
+          if (fileName === '') throw new Error('File Name not Provided:')
+          if (togglestat === 'yes') {
+            fs.writeFileSync(fpath, JSON.stringify(itext))
+          } else {
+            fs.writeFileSync(fpath, itext)
+          }
```
Comment (almeidx) [inline actions/file_control_MOD.js:258]:
```suggestion
          fs.writeFileSync(fpath, togglestat === 'yes' ? JSON.stringify(itext) : itext)
```

File: actions/file_control_MOD.js [hunk 4]
```diff
@@ -226,61 +243,51 @@ module.exports = {
...
+          break
+        case 2: // Append File
+          if (fileName === '') throw new Error('File Name not Provided:')
+          if (togglestat === 'yes') {
+            fs.appendFileSync(fpath, `${JSON.stringify(itext)}\r\n`)
+          } else {
+            fs.appendFileSync(fpath, `${itext}\r\n`)
+          }
```
Comment (almeidx) [inline actions/file_control_MOD.js:266]:
```suggestion
          fs.appendFileSync(fpath, `${togglestat === 'yes' ? JSON.stringify(itext) : itext}\r\n`)
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]

**Pull request merged**